proton-c/src/tests/fuzz/fuzz-sniff-header.c

```diff
@@ -0,0 +1,15 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "core/autodetect.h"
+
+#ifdef __cplusplus
```

Comment:
Thanks, good point.

LubFuzzer does not have a header file. It is a single `.a` library. That is probably why.